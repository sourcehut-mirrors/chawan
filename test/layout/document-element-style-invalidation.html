<!DOCTYPE html>
background should be green
<script>
document.documentElement.getBoundingClientRect()
</script>
<html style="background: green">
