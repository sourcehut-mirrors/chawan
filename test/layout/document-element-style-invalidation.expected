[48;2;0;128;0mbackground should be green                                                      [49m
